guides/common/assembly_supported-usage-and-versions-of-project-components.adoc

```diff
@@ -0,0 +1,32 @@
+include::modules/con_supported-usage-and-versions-of-project-components.adoc[]
+
+ifdef::satellite[]
+include::modules/con_supported-usage-of-project-components.adoc[leveloffset=+1]
+endif::[]
```

Comment:
> Not sure if this is Satellite-only.

Some of the includes are Satellite-only, some are actually non-Satellite. I didn't change these ifdefs in this PR, and to be honest, I'd prefer not to investigate to keep this PR from growing even further.

> You also also shorten this to (for now) one big "if satellite" block.

I prefer smaller ifdef blocks rather than large ones. Because with large ones, there is a risk that someone (like me!) will add a new include in the middle without noticing the ifdefs.

So, no action taken.